Iceberg-Libgit-Filetree/ManifestIcebergLibgitFiletree.class.st

```diff
@@ -8,7 +8,9 @@ Class {
 }
 
 { #category : #'meta-data' }
-ManifestIcebergLibgitFiletree class >> description [ ^ 'I contain the libgit binding for the FileTree export format. 
+ManifestIcebergLibgitFiletree class >> description [
```

Comment:
There isn't a . in first line here, no?

Reply:
the method is just one line.. it was auto-generated. The test I did really just works on the string level "give me the first line, check if there is a .". We should improve that and do it on the AST level... but I think just fixing those methods is not wrong: we should not have methods without a <enter> after the signature in the system, either.

Reply:
+1 thanks